apps/test/netsim/NetSimTable.js

```diff
@@ -518,14 +518,14 @@ describe("NetSimTable", function () {
 
   describe("initial delay coalescing", function () {
     beforeEach(function () {
-      // Re-enable 50ms before-refresh delay to coalesce messages
-      netsimTable.setMinimumDelayBeforeRefresh(50);
+      // Re-enable 100ms before-refresh delay to coalesce messages
+      netsimTable.setMinimumDelayBeforeRefresh(100);
```

Comment:
Consider making this a constant, it seems to be used in a lot of places (presumably the instances of 50 are instances of 100 / 2).